package.json

```diff
@@ -82,8 +82,7 @@
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": [
       "prettier --write",
-      "eslint --fix",
-      "stylelint --fix"
```

Comment:
I think this is a relic from our Linaria days, and we don't need to run stylelint on js/ts 🤔